modules/View.js

```diff
@@ -137,16 +138,87 @@
      * @method prepare
      */
     prepare: function() {
-      if (this.model) {
-        return {
-          model: this.model.toJSON(),
-          view: this.viewState.toJSON()
-        };
-      } else {
-        return {
-          view: this.viewState.toJSON()
-        };
+      return this.__getPrepareFieldsContext();
+    },
+
+    /**
+     * Parses the combined arrays from the defaultPrepareFields array and the prepareFields array (or function
```

Comment:
Briefly give a summary of these options in the prepare method javadoc and link to this doc

Reply:
done